### PR TITLE
Removal of items no longer used on commerical tools admin page

### PR DESCRIPTION
**Pull request by scottp719**
## What does this change?

Removes links on the commercial tools admin page that are no longer used or broken
## What is the value of this and can you measure success?

n/a
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@JonNorman @kelvin-chappell

### CODE REVIEW

File: admin/app/controllers/admin/CommercialController.scala [hunk 1]
```diff
@@ -78,21 +78,6 @@ object CommercialController extends Controller with Logging with AuthLogging wit
...
-    lazy val trailsFuture = ContentApiClient.getResponse(
-      ContentApiClient.search(Edition(request))
-        .pageSize(2)
-    ).map { response  =>
-      response.results.map { item =>
-        FaciaContentConvert.contentToFaciaContent(item)
-      }
-    }
-    trailsFuture map { trails =>
-      NoCache(Ok(views.html.commercial.sponsoredContainers(environment.stage, CommercialPage(), trails)))
```
Comment (kelvin-chappell) [inline admin/app/controllers/admin/CommercialController.scala]: Presumably the `sponsoredContainers` template can go as well.

Comment (ScottPainterGNM) [inline admin/app/controllers/admin/CommercialController.scala]: Good spot!

**Comment by kelvin-chappell** [comment]
👍

File: admin/conf/routes [hunk 2]
```diff
@@ -83,7 +83,6 @@ GET         /commercial/surging
 GET         /commercial/inline-merchandising                                                        controllers.admin.CommercialController.renderInlineMerchandisingTargetedTags()
 GET         /commercial/high-merchandising                                                          controllers.admin.CommercialController.renderHighMerchandisingTargetedTags()
 GET         /commercial/templates                                                                   controllers.admin.CommercialController.renderCreativeTemplates()
-GET         /commercial/sponsored-containers                                                        controllers.admin.CommercialController.sponsoredContainers()
```
Comment (johnduffell) [inline admin/conf/routes]: @ScottPainterGNM you need to take this out of dev-build routes too

Comment (ScottPainterGNM) [inline admin/conf/routes]: Jimmy let me know, just fixing now!

**Pull request merged**